support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -113,27 +96,19 @@ import {
 	paypalOneClickCheckout,
 	setupPayPalPayment,
 } from '../checkout/helpers/paypal';
-import {
-	stripeCreateSetupIntentPrb,
-	stripeCreateSetupIntentRecaptcha,
-} from '../checkout/helpers/stripe';
+import { stripeCreateSetupIntentPrb } from '../checkout/helpers/stripe';
 import {
 	doesNotContainExtendedEmojiOrLeadingSpace,
 	preventDefaultValidityMessage,
 } from '../validation';
 import { BackButton } from './backButton';
 import { CheckoutLayout } from './checkoutLayout';
 import { FormSection, Legend, shorterBoxMargin } from './form';
-import {
-	checkedRadioLabelColour,
-	defaultRadioLabelColour,
-	paymentMethodBody,
-	PaymentMethodRadio,
-	PaymentMethodSelector,
-} from './paymentMethod';
 import { retryPaymentStatus } from './retryPaymentStatus';
 import { setThankYouOrder, unsetThankYouOrder } from './thankYouComponent';
 
+const PaymentSection = lazy(() => import('./paymentSection'));
```

Comment:
Maybe a comment here on why lazy loading used?

Reply:
Yep, if we decided to go with it